lib/galaxy/tools/test.py

```diff
@@ -74,7 +78,7 @@ def description_from_tool_object(tool, test_index, raw_test_dict):
     return ToolTestDescription(processed_test_dict)
 
 
-def _process_raw_inputs(tool, tool_inputs, raw_inputs, required_files, parent_context=None):
+def _process_raw_inputs(tool, tool_inputs, raw_inputs, required_files, required_data_tables, required_loc_files, parent_context=None):
```

Comment:
Someday we need to get this talking to the client and compute less on the backend, I've always really hated that this is using the tool object. One should be able to turn just the list of parameters into an API request without needing Galaxy internals... but I think that is a fight for another day. This is important and we need to just get it done and this is really cool.